aws/dynamodb.go

```diff
@@ -0,0 +1,71 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func getAllDynamoTables(session *session.Session) ([]*string, error) {
+	var TableNames []*string
+	svc := dynamodb.New(session)
+
+	for {
+		result, err := svc.ListTables(&dynamodb.ListTablesInput{})
+		if err != nil {
+			//
+			if aerr, ok := err.(awserr.Error); ok {
+				switch aerr.Error() {
+				case dynamodb.ErrCodeInternalServerError:
+					return nil, errors.WithStackTrace(aerr)
+				default:
+					return nil, errors.WithStackTrace(aerr)
+				}
+			} else {
+				return nil, errors.WithStackTrace(aerr)
+			}
```

Comment:
It seems like in all cases here, you return `errors.WithStackTrace(err)`... So is there a need for the if/else statements?